.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## PR Checklist
+<!-- Please check if your PR fulfills the following requirements: -->
+
+- [ ] Added label to the Pull Request for easier discoverability and search
+- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
```

Comment:
Is PR in this repo the same guideline as the one in main code?

Reply:
This PR template will be used for the design documents and yes we want to apply everything that makes sense from the main repos.

Reply:
most likely if this repo is just documentation then probably only conventional commits `docs:` and `fix:` makes sense IMO